CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 # [Unreleased](https://github.com/purecloudlabs/genesys-cloud-streaming-client/compare/v17.1.0...HEAD)
+### Fixed
+* [PCM-2347](https://inindca.atlassian.net/browse/PCM-2347) - Updated client-logger to fix critical Snyk vulnerability.
```

Comment:
This PR was originally to update to `v4.2.11` of the client-logger but we had another release and another streaming client PR that already updated it to `v4.2.12`. This PR addresses some other Snyk vulns now.